src/main/java/com/datalogics/pdf/samples/manipulation/RemoveInteractivity.java

```diff
@@ -127,6 +128,10 @@ public static void removeInteractivity(final URL inputUrl, final URL outputUrl)
         try {
             pdfDoc = DocumentUtils.openPdfDocument(inputUrl);
 
+            // Before removing any interactive elements, generate their appearances using the AppearanceService
+            // to ensure that the document has all required appearances so the document looks correct at the end.
+            AppearanceService.generateAppearances(pdfDoc, null, null);
```

Comment:
Yes, thanks for looking at this @datalogics-robl. I will add a comment to this to explain it, @datalogics-pgallot and I talked about that as well in person and I just haven't gotten back to this PR yet.